docs/install/_linux.rst

```diff
@@ -7,13 +7,13 @@ Debian/Ubuntu Linux
 Install from PPA repository
 ---------------------------
 
-Use the following commands in Terminal to add the PPA and install Kolibri::
+Use the following commands in Terminal to add the PPA and install Kolibri:
 
-.. code-block:: bash
+	.. code-block:: bash
```

Comment:
These indentations are a bit weird, did your editor add them?

Reply:
They were not rendering properly on RTD...

Before (current on RTD)
![indent](https://user-images.githubusercontent.com/1457929/38102917-9a7af1ca-3339-11e8-878f-26e03caba4ec.png)

After (proposed in the PR)
![indent-after](https://user-images.githubusercontent.com/1457929/38102937-b09fdb32-3339-11e8-8ec7-b36e7bf28cb7.png)

Reply:
Thanks! Got this fixed, it was the indentation that was wrong. You made it work, but with too much indentation :)